lib/psi.rb

```diff
@@ -13,18 +13,21 @@
 # Set 10mm tolerance for edge (thermal bridge) vertices.
 TOL = 0.01
 
-# Sources for the following defaults KHI & PSI values/sets:
+# Sources for thermal bridge types and/or linked default KHI & PSI values/sets:
 
-# BETB = BC Hydro's Building Envelope Thermal Bridging Guide v1.4
+# BETBG = Building Envelope Thermal Bridging Guide v1.4
 # www.bchydro.com/content/dam/BCHydro/customer-portal/documents/power-smart/
 # business/programs/BETB-Building-Envelope-Thermal-Bridging-Guide-v1-4.pdf
```

Comment:
I'll either revise (if needed) the values based on BETBG v1.5, or wait for v1.6.